docs/getting_started/install_manually.md

```diff
@@ -179,40 +140,39 @@ This operation is performed only once when you install eZ Platform for the first
 
     `rm ezp_cron.txt`
 
-## 6. Set up directory permissions
+### Set up permissions
 
 See [Set up directory permissions](set_up_directory_permissions.md) for more information.
 
-## 7. Set up a Virtual Host
+You'll need the web user set as the owner/group on all your files to avoid a 500 error:
+
+``` bash
+chown -R www-data:www-data /var/www/ezplatform
+```
+
+### Set up Virtual Host
 
 This example demonstrates using Apache2 as part of the traditional LAMP stack.
 
-### Option A: Scripted Configuration
+#### Scripted configuration
```

Comment:
A and B made sense here, because these the two alternative options, not sequential steps.